src/main/java/org/kuali/kfs/pdp/businessobject/ACHBank.java

```diff
@@ -0,0 +1,254 @@
+/**
+ * The Kuali Financial System, a comprehensive financial management system for higher education.
+ *
+ * Copyright 2005-2019 Kuali, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.kuali.kfs.pdp.businessobject;
+
+import org.kuali.kfs.krad.bo.PersistableBusinessObjectBase;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.kfs.sys.businessobject.PostalCode;
+import org.kuali.kfs.sys.businessobject.State;
+import org.kuali.rice.core.api.mo.common.active.MutableInactivatable;
+
+public class ACHBank extends PersistableBusinessObjectBase implements MutableInactivatable {
+
+    protected String bankRoutingNumber;
+    protected String bankOfficeCode;
+    protected String bankServiceNumber;
+    protected String bankTypeCode;
+    protected String bankNewRoutingNumber;
+    protected String bankName;
+    protected String bankStreetAddress;
+    protected String bankCityName;
+    protected String bankStateCode;
+    protected String bankCountryCode;
+    protected String bankZipCode;
+    protected String bankPhoneAreaCode;
+    protected String bankPhonePrefixNumber;
+    protected String bankPhoneSuffixNumber;
+    protected String bankInstitutionStatusCode;
+    protected String bankDataViewCode;
+    protected boolean active;
+
+    protected State bankState;
+    protected PostalCode postalCode;
+
+    public ACHBank() {
+        // we need country code to have a value for OJB foreign keys to other location types but it isn't exposed on
+        // docs so it never gets set. setting a default value on the column in the db did not do what we want b/c
+        // the ojb insert explicitly specifies every column it knows about. this will work for now.
+        bankCountryCode = KFSConstants.COUNTRY_CODE_UNITED_STATES;
+    }
+
+    /**
+     * This constructor takes a line of data from FedACH Directory file and populates the ACHBank object
+     *
+     * @param fileData line of data from the FedACH Directory file
+     */
+    public ACHBank(String fileData) {
```

Comment:
The original file appears to have a call to "setBankDataViewCode" in this method, but I don't see that call in this overlay. Can you please add that back in if necessary?

Reply:
Good catch. Thanks.